src/adapters/node-http/monkeyPatch.ts

```diff
@@ -1,20 +1,20 @@
 import { z } from 'zod';
 
-import { OpenApiProcedureRecord, OpenApiRouter } from '../../types';
-import { forEachOpenApiProcedure, getInputOutputParsers } from '../../utils/procedure';
+import { OpenApiProcedure } from '../../types';
+import { getInputOutputParsers } from '../../utils/procedure';
 import { instanceofZodType, instanceofZodTypeLikeVoid } from '../../utils/zod';
 
-export const monkeyPatchVoidInputs = (appRouter: OpenApiRouter) => {
-  const { queries, mutations } = appRouter._def;
-  const zObject = z.object({});
-
-  const voidInputPatcher = (procedure: OpenApiProcedureRecord[string]) => {
-    const { inputParser } = getInputOutputParsers(procedure);
-    if (instanceofZodType(inputParser) && instanceofZodTypeLikeVoid(inputParser)) {
+export const monkeyPatchProcedure = (procedure: OpenApiProcedure) => {
+  if ((procedure as any).__monkeyPatched) {
+    return;
+  }
+  (procedure as any).___monkeyPatched = true;
```

Comment:
No need to double monkey patch (this will only happen on standalone/express servers anyway).